internal/server/middleware/grpc/middleware.go

```diff
@@ -566,3 +568,45 @@ func (e evaluationCacheKey[T]) Key(r T) (string, error) {
 
 	return fmt.Sprintf("%s:%s:%s:%s", string(e), r.GetFlagKey(), r.GetEntityId(), out), nil
 }
+
+// x-flipt-server-version represents the maximum version of the flipt server that the client can handle.
+const fliptServerVersionHeaderKey = "x-flipt-server-version"
```

Comment:
wondering if i should make this more clear as a header key, like `X-Flipt-Accept-Server-Version`?

Reply:
I like your thinking here calling it accept server version. Feels more intentional.